store/tikv/mock-tikv/mvcc.go

```diff
@@ -130,18 +130,18 @@ func (e *mvccEntry) Prewrite(mutation *kvrpcpb.Mutation, startTS uint64, primary
 	return nil
 }
 
-func (e *mvccEntry) checkTxnCommitted(startTS uint64) (uint64, bool) {
+func (e *mvccEntry) getTxnCommitInfo(startTS uint64) (*mvccValue, bool) {
```

Comment:
Since the first argument is a pointer, it doesn't need to return the second value.
A nil `*mvccValue` means not found.